### PR TITLE
[maskedtensor] port torch/_masked into torch/masked

### PR DESCRIPTION
**Pull request by george-qi**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #85515

### CODE REVIEW

File: torch/masked/_docs.py [hunk 1]
```diff
@@ -587,7 +587,7 @@
     >>> mask
     tensor([[ True, False,  True],
             [False, False, False]])
-    >>> torch._masked.mean(input, 1, mask=mask)
+    >>> torch.masked._ops.mean(input, 1, mask=mask)
```
Comment (cpuhrsch) [inline torch/masked/_docs.py:590]: A good follow up would be to update the documentation and use a `MaskedTensor`

**Review by cpuhrsch — APPROVED** [review]

**Comment by mehtanirav** [comment]
@george-qi  We need to revert this PR as a couple of PRs below it need to be reverted. Please re-create these changes in a separate PR that can be merged.  Sorry for the inconvenience.

**Pull request closed without merging**